app/scripts/lib/app-start.js

```diff
@@ -60,6 +60,7 @@ define([
   'models/auth_brokers/iframe',
   'models/unique-user-id',
   'models/user',
+  'models/verification/same-browser',
```

Comment:
@vladikoff, @philbooth - any suggestions on a better name? The idea I'm trying to convey is:

"Verification info that is used if the user verifies in the same browser."